packages/core/src/utils/config.ts

```diff
@@ -246,7 +246,7 @@ export function useReactiveResponsiveConfig(
 }
 
 export function setComponentThemeConfig(
-  cmp: { TAG_NAME: string } | string,
+  cmp: { TAG_NAME: string } | string | any,
```

Comment:
why any here?

Reply:
Linter complaining and didn't find a good model match